addons/account_payment_term/models/account_payment_term.py

```diff
@@ -40,15 +40,5 @@ def _get_due_date(self, date_ref):
 
         due_date = fields.Date.from_string(date_ref) or fields.Date.today()
         if self.delay_type == 'days_end_of_month_on_the':
-            date_end_of_month = date_utils.end_of(due_date + relativedelta(days=self.nb_days), 'month')
```

Comment:
I think `date_utils` is not used anymore

Reply:
nice catch